.github/workflows/windows.yml

```diff
@@ -0,0 +1,21 @@
+name: Windows build
+
+on: push, pull_request
+
+env:
+  # Customize the env if
+  BUILD_TYPE: Release
+  DPDK_VERSION: 22.07
+
+jobs:
+  Build:
+    #if: github.repository == 'OpenVisualCloud/Media-Transport-Library'
```

Comment:
Pls enable this check since self-host can only run for OpenVisualCloud/Media-Transport-Library repo